From 23d0b5f163b39704d3876d7973bb2e249dfe9ef8 Mon Sep 17 00:00:00 2001
From: Mahyar Koshkouei <mahyar.koshkouei@gmail.com>
Date: Mon, 30 Oct 2017 18:53:33 +0000
Subject: [PATCH 1/1] Remove locale support by default for Unix

mgba will not compile if the toolchain does not support locale features.

Signed-off-by: Mahyar Koshkouei <mahyar.koshkouei@gmail.com>
---
 Makefile.libretro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.libretro b/Makefile.libretro
index ed48a93..24244f5 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -81,7 +81,7 @@ ifneq (,$(findstring unix,$(platform)))
    TARGET := $(TARGET_NAME)_libretro.so
    fpic := -fPIC
    SHARED := -shared -Wl,-version-script=link.T
-   DEFINES += -std=c99 -D_GNU_SOURCE -DHAVE_LOCALE
+   DEFINES += -std=c99 -D_GNU_SOURCE
 
    # Raspberry Pi 3
    ifneq (,$(findstring rpi3,$(platform)))
-- 
2.14.2

